### DISCUSSION TITLE
Instructions / Tutorial for setting up for boot project

### DISCUSSION THREAD

**Discussion by wontheone1** [Day8/re-frame-10x#295] [Q&A] [answered]
Hello ! I have a boot project that I want to use 'reframe' and 're-frame-10x' for it.
I mostly use leiningen previously so not so good with boot tasks.

I guess boot should be able to use this with some tasks but it would be great to see some instructions / tutorials on how to use re-frame-10x in boot project.
Or just example boot task definition.

**Comment by danielcompton** [comment]
I don’t think there is anything Boot specific here, you will just need to add the dependencies and update your compiler options. If anyone wants to contribute docs though, that’d be welcome.

**Comment by netpyoung** [comment, marked as answer]
I found template for boot https://github.com/robjens/boot-reframe-10x

**Comment by wontheone1** [comment]
@netpyoung @danielcompton thank you all ! I'll take a look at the template when I need it.
